src/sass/partials/_global.scss

```diff
@@ -0,0 +1,55 @@
+body {
+  font-family: $fontPrimary;
+  color: $IIColor;
+  background-color: $IColor;
+  font-size: 14px;
+  letter-spacing: 0.03em;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+ul {
+  list-style: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+address {
+  font-style: normal;
+}
+
+a {
+  text-decoration: none;
+}
+
+.container {
+  width: 100%;
+  padding: 0 15px;
```

Comment:
padding: 0 20px in container

Reply:
I don't write this code just move it from main to partial scss, because this code here overrides changes in other files. This code belongs to you and was added in PR https://github.com/VasylDolishnii/goit-MainMasters/pull/4.